src/main/frontend/state.cljs

```diff
@@ -131,6 +131,7 @@
      ;; whether database is persisted
      :db/persisted?                         {}
      :cursor-range                          nil
+     :editing-input?                        false
```

Comment:
Better rename this state variable to a more meaningful one, with a namespace or something.
like: `:editor/auto-completing?`